src/appengine/handlers/cron/fuzzer_weights.py

```diff
@@ -317,6 +315,22 @@ def update_target_weights_for_engine(client, engine, specifications):
   logs.log('Weight adjustments complete for engine %s.' % engine)
 
 
+def store_current_weights_in_bigquery():
+  """Update a bigquery table containing the daily stats."""
+  rows = []
+  target_jobs = ndb_utils.get_all_from_model(data_types.FuzzTargetJob)
+  for target_job in target_jobs:
+    row = {
+        'fuzzer': target_job.fuzz_target_name,
+        'job': target_job.job,
+        'weight': target_job.weight
+    }
+    rows.append(big_query.Insert(row, None))
```

Comment:
row=row, insert_id=None

Also, is it ok to ignore insert_id ?

Reply:
Done. insert_id is an optional field, so this should be fine. In testing it seemed like all of the rows were inserted properly.